vibin/mediaservers/asset.py

```diff
@@ -1,3 +1,4 @@
+import pathlib
```

Comment:
Since we already have `from pathlib import Path`, this `import pathlib` can be removed -- although you'll need to change the `pathlib.Path()` call on line 246 to just `Path()`.

Reply:
Done - sorry, missed that.